packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -617,22 +618,80 @@ class _ConstraintsDescriptorState extends State<ConstraintsDescriptor>
 
 
 class StoryOfYourFlexWidget extends StatelessWidget {
-  const StoryOfYourFlexWidget(InspectorTreeNode node, {
+  StoryOfYourFlexWidget(this.node, {
     Key key,
   }) : super(key: key);
 
+  InspectorTreeNode node;
+
   @override
   Widget build(BuildContext context) {
+    final Map<String, Object> flexDetails = node.diagnostic?.json['flex'];
```

Comment:
Exciting to see this!
Go ahead and add getters for the extra json fields you add to RemoteDiagnosticsNode.